src/ui/profile_preferences.ui

```diff
@@ -2153,6 +2153,19 @@ you can use it but there could be issues with aligning columns of text</string>
             </layout>
            </widget>
           </item>
+          <item row="1" column="0">
+           <widget class="QCheckBox" name="mMapperShowBorders">
```

Comment:
Might I suggest a rename to `checkBox_mapperShowBorders` or even `checkBox_mapperShowRoomBorders` ?

Reply:
Sure, this is very reasonable thing to do.